CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.3.0 (2024-03-12)
+
+### Changes
+
+* Migrate to Vue 3.
```

Comment:
Since this is a minor release, we did not migrate — it is still compatible with both.

So say:

"Compatible with both Apostrophe 3.x and Apostrophe 4.x (both Vue 2 and Vue 3)"